docs/templates/explanation-template.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Noun or nouns describing your subject
+content_type: explanation
+no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+beta: true # Labels the page as beta; adds a banner to the top of the page.
+disable_image_expand: true # Stops images from expanding in a modal on click. Sets it for the entire page.
+class: no-copy-code # Disables the copy code button for all code blocks on the page.
```

Comment:
```suggestion

# Optional values. Uncomment any that apply to your document.

# no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
# alpha: true # Labels the page as alpha; adds a banner to the top of the page.
# beta: true # Labels the page as beta; adds a banner to the top of the page.
# disable_image_expand: true # Stops images from expanding in a modal on click. Sets it for the entire page.
# class: no-copy-code # Disables the copy code button for all code blocks on the page.
```
Similar to the plugin template, how about commenting out any values that are not required in the front matter? Not sure if the intro line I added is necessary though.